docs/gc.md

```diff
@@ -132,15 +132,35 @@ There are a number of advantages to using arraylets.
 
-[`-Xgcpolicy:optavgpause`](xgcpolicy.md#optavgpause) uses concurrent mark and sweep phases, which means that pause times are reduced when compared to optthruput, but at the expense of some performance throughput.
+The realtime heap is allocated as a contiguous range of RAM partitioned into small regions of equal size, usually about 64 KB. 
 
-### `optthruput` policy
+As with the `balanced` GC policy, arrays are represented as arraylets with a spine pointing to a series of regions containing the array elements (see [Arraylets: dealing with large arrays](#arraylets-dealing-with-large-arrays) for more information). Note that JNI access to array data might involve reconstituting arraylets as contiguous arrays, which can significantly slow down processing.
 
-[`-Xgcpolicy:optthruput`](xgcpolicy.md#optthruput) is optimized for throughput by disabling the concurrent mark phase, which means that applications will stop for long pauses while garbage collection takes place. You might consider using this policy when high application throughput, rather than short garbage collection pauses, is the main performance goal.
+Each region of the heap is either empty or contains only objects of the same size or an arraylet. This organization simplifies new object allocation and consolidation of free heap space, allowing the realtime collector to maintain GC throughput while supporting a consistent service level.
+
+Although high application utilization is desirable for optimal throughput, the GC must be able to keep up with the application's memory  allocation rate. 
+
+A higher utilization typically requires a larger heap because the GC isn't allowed to run as much as a lower utilization would permit. The relationship between utilization and heap size is highly application dependent, and striking an appropriate balance requires iterative experimentation with the application and VM parameters. You might need to adjust heap size or pause time or target utilization to achieve an acceptable runtime configuration.
+
+See [`-Xgcpolicy:metronome`](xgcpolicy.md#metronome-aix-linux-x86-only) for details of the configuration parameters you can set.
+
+### `optavgpause` and `optthruput` policies
+
+Both of these policies use the global collector to manage a flat heap comprised of a single region. The application runs until the heap is exhausted and the global collector runs to mark and sweep the heap. Additionally, if the heap becomes fragmented, the global collector compacts the heap.
+
+These policies are best suited to short-lived applications and to long-running services involving concurrent sessions that have short lifespans. Short-lived applications with adequate heap sizes usually complete without compaction, and the flat heap fragments more slowly when session-bound objects are allocated and drop out of the live set in short overlapping clusters.
```

Comment:
In reality these two policies are rather inferior to gencon and it's hard to find a scenario where they would really applicable. I'll elaborate a bit for hopefully better general understanding, although I'm not sure how much of it we really want to document.

The best advantage of optthruput is that it's free of any object access barriers (unlike optavgpause tjat has write barrier to support concurrent mark, and gencon which in addition to that one also has a generational write barrier). In some workloads, cost of those barriers may be high enough to outweigh the other benefits of concurrent and generational aspect (like low pauses and high yield of free memory).

To some extent we could say that for optavgpause, but applying only to the lack of generational barrier. So, for example worklaods that frequently change large and old reference arrays may have costly generational barrier, since those array have to be scanned as part of root set for Scavenge.

An additional potential problem with generation GCs is that it splits heap so that a very large object in worst case might not fit in either of the 2 areas, while there is enough total free memory, or in better case, allocation may succeed but require additional GC cycles and copying of objects (relative to non-generational GC) to accommodate large enough contiguous free space.

Reply:
I've changed the docs and incorporated some of the information above. Does it now make sense? Is it accurate? Have I said enough / too much?!